### DISCUSSION TITLE
email-template to pick Localization file from external storage (e.g. from a MongoDB database)

### DISCUSSION THREAD

**Discussion by agrawalrohant** [forwardemail/forwardemail.net#257] [General]
Hi, Using email-templates library is it possible to pick Localization file from external storage (e.g. from a MongoDB database) just like we can pick the html files from an external storage ?
If yes, what are the hooks available to handle this?

**Comment by titanism** [comment]
Not supported at the moment but a PR is welcome.

See code at <https://github.com/ladjs/i18n> (functions are synchronous, not async) due to Pug rendering namely being sync.

**Reply by agrawalrohant** [reply]
ok.Thanks. How about handlebars, can this be supported with this? I mean with externalising the localisation files ?
